packages/core/admin/server/content-types/index.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   user: { schema: require('./User') },
   role: { schema: require('./Role') },
   'api-token': { schema: require('./api-token') },
+  'token-permission': { schema: require('./token-permission') },
```

Comment:
Just a thought but shouldn't we call this `api-token-permission` to keep consistency with `api-token`?